### PR TITLE
fix typos and optimize autotest part in README

### PR DESCRIPTION
**Pull request by kevinwenminion**
1. README is updated especially for the autotest part.
2. We change the default deformation parameters in Elastic to 1e-2 which is more reasonable for calculating elastic constants.

### CODE REVIEW

**Comment by HuangJiameng** [comment]
What do you think about the connection between README and wiki on autotest? Could you help to combine their content into one document and put it on [DPGEN’s documentation](docs.deepmodeling.com/projects/dpgen/en/devel/)?

**Review by wanghan-iapcm — APPROVED** [review]

**Comment by wanghan-iapcm** [comment]
> What do you think about the connection between README and wiki on autotest? Could you help to combine their content into one document and put it on [DPGEN’s documentation](docs.deepmodeling.com/projects/dpgen/en/devel/)?

This PR intends to replace the autotest doc in the wiki. All information in the wiki is supposed to be moved to README

File: README.md [hunk 1]
```diff
@@ -688,300 +694,369 @@ the following essential section should be provided in user template
...
+`init_data_path` is the path of VASP or LAMMPS initial data to be reproduced. `init_from_suffix` is the suffix of the initial data and the default value is "00". In this case, the VASP Interstitial results are stored in `../vasp/confs/std-*/interstitial_00` and the reproduced Interstitial results would be in `deepmd/confs/std-*/interstitial_reprod`. 
+
+`reprod_last_frame` denotes if only the last frame is used in reproduce. The default value is True for eos and surface, but is False for vacancy and interstitial.
+
+Three steps for reproduce calculations are also `make`, `run`, and `post`.
 
-### The content of the auto_test
-To know what actually will dpgen autotest do, including the lammps and vasp script, the input file and atom configuration file auto_test will generate, please refer to https://hackmd.io/@yeql5ephQLaGJGgFgpvIDw/rJY1FO92B
+### More Information of Autotest
+For more information of autotest, please refer to [dpgen wiki](https://github.com/deepmodeling/dpgen/wiki).
```
Comment (HuangJiameng) [inline README.md:1059]: > This PR intends to replace the autotest doc in the wiki. All information in the wiki is supposed to be moved to README

But the wiki is cited here. Should we keep the wiki separate or move it to the documentation site?

Comment (wanghan-iapcm) [reply README.md:1059]: @kevinwenminion plz move it to the documentation site

**Review by pxlxingliang — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HuangJiameng) [reply README.md:1059]: https://github.com/deepmodeling/dpgen/pull/891 this pr move the wiki to the documentation site

**Review by HuangJiameng — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kevinwenminion) [reply README.md]: I have removed the citation of wiki in README. Users can learn how to use autotest very quickly by reading the updated autotest part in README.

**Pull request merged**